permissions/Audio.permission

```diff
@@ -6,6 +6,9 @@
 # x-sailjail-translation-key-long-description = permission-la-audio_description
 # x-sailjail-long-description = Play audio and show audio controls on lockscreen
```

Comment:
Since this will give access to microphone, should we mention here that as well? From user perspective that would be nice to know.

For developers we could move that Microphone to the public list and add description there. Also might be useful to describe somewhere (perhaps as a comment in Audio.permission if anyone reads those) that while currently Audio gives access to microphone, developer should require Microphone for compatibility with future releases.

Reply:
Generally this gives access to any audio Pulseaudio can handle. Since we have in mind to split the permissions might it be more clearer for the future if we kept the descriptions separate and have only as a comments in the permission files to describe the actual functionality.

However, I see the point that user might want to know that by giving this `Audio` permission to an application it can also access microphone. Perhaps we should change the descriptions when the functionality changes, and also the translations need to be updated then.

Reply:
Yes, exactly what I meant. Let's change the descriptions again when the functionality changes. And because the intention of the permission is different from what it actually does, we should document that in the comments so that developers can prepare for that change happening later.

Reply:
@Tomin1 updated these in 3c6c421 - using "Microphone" in the descriptions might be too long even? Since we've used simply "play audio" I thought that "record audio" might be better in line with that.

"Play audio and record with microphone, and show audio controls ..." seems quite long for a description even.

Reply:
Yeah, no need to mention that microphone. I think the way you wrote long description is good.